headers/headers.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2020 YourBase Inc.
+// SPDX-License-Identifier: BSD-3-Clause
+
+// Package headers provides constants for well-known HTTP headers.
+package headers
```

Comment:
http/headers? Just because that's what they are?

Reply:
Done.